src/player.ts

```diff
@@ -1808,6 +1808,7 @@ export default class Player extends FakeEventTarget {
       const classNameWithId = `${ENGINE_CLASS_NAME}-${this._engine.id}`;
       Utils.Dom.addClassName(engineEl, classNameWithId);
       Utils.Dom.prependTo(engineEl, this._el);
+      Utils.Dom.setAttribute(engineEl, 'tabindex', '-1');
```

Comment:
why does this fix the problem ?

Reply:
Firefox treats <video> as a focusable element (don't know why but it happens in all video elements) if you will add tabIndex="-1" so explicitly it is not focusable element.